aten/src/ATen/cpu/vec/vec512/vec512_complex_double.h

```diff
@@ -438,17 +438,24 @@ template <> Vectorized<c10::complex<double>> inline operator*(const Vectorized<c
 template <> Vectorized<c10::complex<double>> inline operator/(const Vectorized<c10::complex<double>> &a,
                                                              const Vectorized<c10::complex<double>> &b) {
   //re + im*i = (a + bi)  / (c + di)
-  //re = (ac + bd)/abs_2()
-  //im = (bc - ad)/abs_2()
-  const __m512d sign_mask = _mm512_setr_pd(-0.0, 0.0, -0.0, 0.0, -0.0, 0.0, -0.0, 0.0);
-  auto ac_bd = _mm512_mul_pd(a, b);         //ac       bd
+  auto mask = _mm512_set1_pd(-0.f);
+  auto fabs_cd = _mm512_andnot_pd(mask, b);     // |c|    |d|
+  auto fabs_dc = _mm512_permute_pd(fabs_cd, 0x55);   // |d|    |c|
+  auto scale = _mm512_max_pd(fabs_cd, fabs_dc);  // sc     sc
```

Comment:
Why aren't you using `_mm512_rcp14_p{s,d}` for AVX512?

Reply:
Reading at the documentation for rcp14, it says the error is bounded by 2^-14 which is about 6e-5 and it’s too high in my opinion

Reply:
The point here is that you don't need the exact number, an approximation is alright, as `(ad)/(bd) = a/b`. cf. https://github.com/pytorch/pytorch/pull/93277#discussion_r1092420699

Reply:
Yes, you’re right. I’ll make a new push once I get back home